Podcast/SearchAllEndpointRequest.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  SearchAllEndpointRequest.swift
+//  Podcast
+//
+//  Created by Kevin Greer on 3/14/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+import SwiftyJSON
+
+class SearchAllEndpointRequest: EndpointRequest {
+    
+    init(query: String, offset: Int, max: Int) {
```

Comment:
add parameters as class properties when you make endpoint requests 

not super necessary, but its a useful convention we should maintain